src/OctoshiftCLI.IntegrationTests/TestHelper.cs

```diff
@@ -585,5 +585,15 @@ public async Task AssertBoardsIntegrationConfigured(string adoOrg, string teamPr
             boardsConnection.Should().NotBeNull();
             boardsConnection.repoIds.Count().Should().Be(1);
         }
+
+        public async Task AssertMigrationLogFileExists(string githubOrg, string repo)
+        {
+            _output.WriteLine("Checking that the migration log was downloaded...");
+
+            var migrationLogFile = $"migration-log-{githubOrg}-{repo}.log";
+            var migrationLogExists = await Task.Run(() => File.Exists(migrationLogFile));
+
+            migrationLogExists.Should().Be(true);
```

Comment:
This can be simplified
```suggestion
            File.Exists(migrationLogFile).Should().Be(true);
```

Reply:
I used `Task.Run` here because of this warning: 

> This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.

I added the async `Task.Run` in https://github.com/github/gh-gei/pull/382/commits/ca9e42928c70c02763bed76f6bbc9866e2e73337 in a response to this CI failure:

https://github.com/github/gh-gei/runs/6503637347?check_suite_focus=true

There isn't an async equivalent of `File.Exists` or similar, so I'm wrapping it in a `Task.Run` here :slightly_smiling_face: 

What are your thoughts?

Reply:
When this is closer to ready (when prod is updated and int tests are passing) we can pair on this to fix it. But the error is basically saying that you declared this method as async (AssertMigrationLogFileExists), but you're not doing any async stuff in it.  You can fix that in 2 ways, either do some async stuff (which is what you tried to do), or just don't declare the method as async, which I  think is what you should do.

Reply:
two things:
1. def. don't need to use `Task.Run` 
2. nit: there is a neater syntax for checking booleans: `Should().BeTrue()`